templates/controllers/tab/settings/archiving/form/archivingForm.tpl

```diff
@@ -12,49 +12,77 @@
-		{/fbvFormSection}
-	{/fbvFormArea}
-
-	<div class="lockss_description">
-		<h3>{translate key="manager.setup.lockssTitle"}</h3>
-		<p>
-			{translate key="manager.setup.lockssDescription"}
+	{if $isPLNPluginInstalled}
+		{fbvFormArea id="mainLockss"}
+			{fbvFormArea title="manager.setup.enablePLNArchive" id="plnArea"}
+				{fbvFormSection list="true" translate=false}
+					{translate|assign:"enablePLNArchivingLabel" key="manager.setup.plnPluginEnable"}
+					{fbvElement type="checkbox" id="enablePln" value="1" checked=$isPLNPluginEnabled label=$enablePLNArchivingLabel translate=false}
+				{/fbvFormSection}
+			{/fbvFormArea}
+			{if $isPLNPluginEnabled}
+				{url|assign:depositsGridUrl component="plugins.generic.pln.controllers.grid.PLNStatusGridHandler" op="fetchGrid" escape=false}
+				{load_url_in_div id="depositsGridContainer" url=$depositsGridUrl}
+			{/if}
+		{/fbvFormArea}
+		<p class="expand-others">
+			<a id="toggleOthers" href="#">{translate key="manager.setup.otherLockss"}</a>
 		</p>
-		<p>
-			{url|assign:"lockssExistingArchiveUrl" router=$smarty.const.ROUTE_PAGE page="user" op="email" template="LOCKSS_EXISTING_ARCHIVE"}
-			{url|assign:"lockssNewArchiveUrl" router=$smarty.const.ROUTE_PAGE page="user" op="email" template="LOCKSS_NEW_ARCHIVE"}
-			{translate key="manager.setup.lockssRegister" lockssExistingArchiveUrl=$lockssExistingArchiveUrl lockssNewArchiveUrl=$lockssNewArchiveUrl}
-		</p>
-	</div>
+	{else}
+		<span>
+			{translate key="manager.setup.plnPluginNotInstalled"}
+		</span>
+	{/if}
+	
+	{fbvFormArea id="otherLockss"}
+		{fbvFormArea title="manager.setup.enableArchive" id="lockssArea"}
+			{fbvFormSection list="true" translate=false}
+				{url|assign:"lockssUrl" router=$smarty.const.ROUTE_PAGE page="gateway" op="lockss"}
+				{translate|assign:"enableLockssLabel" key="manager.setup.lockssEnable" lockssUrl=$lockssUrl}
+				{fbvElement type="checkbox" id="enableLockss" value="1" checked=$enableLockss label=$enableLockssLabel translate=false}
 
-	<div class="clockss_description">
-		<h3>{translate key="manager.setup.clockssTitle"}</h3>
-		<p>
-			{translate key="manager.setup.clockssDescription"}
-		</p>
-		<p>
-			{translate key="manager.setup.clockssRegister"}
-		</p>
-	</div>
+				{url|assign:"clockssUrl" router=$smarty.const.ROUTE_PAGE page="gateway" op="clockss"}
+				{translate|assign:"enableClockssLabel" key="manager.setup.clockssEnable" clockssUrl=$clockssUrl}
+				{fbvElement type="checkbox" id="enableClockss" value="1" checked=$enableClockss label=$enableClockssLabel translate=false}
+			{/fbvFormSection}
+		{/fbvFormArea}
+
+		<div class="lockss_description">
+			<h3>{translate key="manager.setup.lockssTitle"}</h3>
+			<p>
+				{translate key="manager.setup.lockssDescription"}
+			</p>
+			<p>
+				{url|assign:"lockssExistingArchiveUrl" router=$smarty.const.ROUTE_PAGE page="user" op="email" template="LOCKSS_EXISTING_ARCHIVE"}
```

Comment:
Do these links still work? I don't recall OTOH.

Reply:
It seems that the URL is not pointing to anything. I am waiting for the texts update from Mark, I think that there will be updates to that too. If not, I will ask Mark on that.